_activities/gravity.md

```diff
@@ -4,12 +4,22 @@ layout: activity
 # Content
 #
 title: "Gravity"
-title_fr: ""
+title_fr: "Gravité"
 age: 8-12
-objective: ""
-objective_fr: ""
-summary: ""
-summary_fr: ""
+objective: "Students will learn how to quantitatively describe how things fall under the affect of gravity. "
+objective_fr: "Les élèves vont apprendre à décrire quantitativement comment que les objets tombent sous l'éffet de la gravité."
+summary: "Students will drop a ball bearing into a small box that is filled with sand. This experiment will be performed twice; the first time the ball will have no initial velocity, and the second time the ball will have an initial velocity. Students will then use a flashlight to observe the craters made in the sand, and discuss the differences between the two craters."
```

Comment:
We actually have students drop ball bearings two at a time comparing three different things (they will do three different experiments): (1) two balls at the same height but with different mass, (2) two balls with the same mass dropped from different heights, and (3) two balls with the same mass and dropped from similar height, but one is thrown instead of dropped. Can this be revised to reflect this procedure instead?